tutor/src/screens/assignment-edit/ux.js

```diff
@@ -217,6 +222,11 @@ export default class AssignmentUX {
     return <Step ux={this} />;
   }
 
+  @action.bound renderMiniCloneEditor(form) {
+    this.form = form;
+    return <DetailsBody ux={this} />;
+  }
+
```

Comment:
This seems dangerous because rendering shouldn't set properties, but I don't see where this is called from.  Beside being bad form, React & Mobx will sometimes complain about circular references if setting `form` causes a re-render.